<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Hanuma" failures="2" tests="2" name="Test(failed)(failed)" time="21.851" errors="0" timestamp="2024-08-02T17:11:41 IST">
  <testcase classname="testCases.TC03_LoginDataDrivenTest" name="verify_LoginDDT" time="11.442">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.TC03_LoginDataDrivenTest.verify_LoginDDT(TC03_LoginDataDrivenTest.java:43)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- verify_LoginDDT -->
  <testcase classname="testCases.TC03_LoginDataDrivenTest" name="verify_LoginDDT" time="2.64">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.TC03_LoginDataDrivenTest.verify_LoginDDT(TC03_LoginDataDrivenTest.java:51)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- verify_LoginDDT -->
</testsuite> <!-- Test(failed)(failed) -->
